docs/root/version_history/current.rst

```diff
@@ -104,7 +104,7 @@ Removed Config or Runtime
 
 New Features
 ------------
-
+* access log: added a new response flag for upstream cluster not found.
```

Comment:
Maybe be a bit more explicit in what this is.

Reply:
Added. What else do I need?